howso/hyperparameters.amlg

```diff
@@ -512,70 +517,35 @@
 				context_key_options
 					(sort
 						(lambda (> (size (current_value)) (size (current_value 1))))
-						(indices (get !hyperparameterMetadataMap feature) )
+						(indices (get !hyperparameterMetadataMap target_mode) )
```

Comment:
If this returns the first candidate context key it finds to be suitable, then perhaps we should instead sort the keys in *decreasing* length. Maybe it already does that and the comment is wrong, I don't remember exactly how the lambda in sort behaves